roles/osquery/README.rst

```diff
@@ -0,0 +1,35 @@
+This ansible role installs osquery.
+
+**Role Variables**
+
+.. zuul:rolevar:: osquery_service_name
+   :default: osqueryd
+
+Service name for osquery.
+
+.. zuul:rolevar:: osquery_package_name
+   :default: osquery
+
+The name of the package for osquery.
+
+.. zuul:rolevar:: osquery_configure_repository
+   :default: true
+
+Prepare the system for installing osquery. Add the repository key
+and the repository itselfs.
+
+.. zuul:rolevar:: osquery_debian_repository_arch
+   :default: amd64
+
+Architecture of the target system.
```

Comment:
Shouldn't ansible be able to automatically detect this?

Reply:
Probably possible. Let's do this in a separate PR and keep it like everywhere.